SecurityPkg/Include/Library/Tpm2CommandLib.h

```diff
@@ -1057,173 +1057,6 @@ Tpm2ReadPublic (
-
-  @param HashAlg            Hash algorithm to be checked.
-  @param HashAlgorithmMask  Bitfield of allowed hash algorithms.
-
-  @retval TRUE  Hash algorithm is supported.
-  @retval FALSE Hash algorithm is not supported.
-**/
-BOOLEAN
-EFIAPI
-IsHashAlgSupportedInHashAlgorithmMask (
-  IN TPMI_ALG_HASH  HashAlg,
-  IN UINT32         HashAlgorithmMask
-  );
-
-/**
-  Copy TPML_DIGEST_VALUES into a buffer
-
-  @param[in,out] Buffer             Buffer to hold copied TPML_DIGEST_VALUES compact binary.
-  @param[in]     DigestList         TPML_DIGEST_VALUES to be copied.
-  @param[in]     HashAlgorithmMask  HASH bits corresponding to the desired digests to copy.
-
-  @return The end of buffer to hold TPML_DIGEST_VALUES.
-**/
-VOID *
-EFIAPI
-CopyDigestListToBuffer (
-  IN OUT VOID            *Buffer,
-  IN TPML_DIGEST_VALUES  *DigestList,
-  IN UINT32              HashAlgorithmMask
-  );
-
-// MU_CHANGE - START - [TCBZ3267] Additional helper functions.
-
-/**
-  Copy a buffer into  TPML_DIGEST_VALUES structure.
-  This is the opposite to the CopyDigestListToBuffer function.
-
-  @param[in]     Buffer             Buffer to hold TPML_DIGEST_VALUES compact binary.
-  @param[in]     BufferSize         Size of Buffer.
-  @param[in,out] DigestList         TPML_DIGEST_VALUES.
-
-  @return EFI_STATUS
-  @retval EFI_SUCCESS               Buffer was successfully copied to Digest List.
-  @retval EFI_BAD_BUFFER_SIZE       Bad buffer size passed to function.
-  @retval EFI_INVALID_PARAMETER     Invalid parameter passed to function: NULL pointer or
-                                    BufferSize bigger than TPML_DIGEST_VALUES
-**/
-EFI_STATUS
-EFIAPI
-CopyBufferToDigestList (
-  IN     VOID                *Buffer,
-  IN     UINT32              BufferSize,
-  IN OUT TPML_DIGEST_VALUES  *DigestList
-  );
-
-// MU_CHANGE - END - [TCBZ3267] Additional helper functions.
-
-/**
-  Get TPML_DIGEST_VALUES data size.
-
-  @param[in]     DigestList    TPML_DIGEST_VALUES data.
-
-  @return TPML_DIGEST_VALUES data size.
-**/
-UINT32
-EFIAPI
-GetDigestListSize (
-  IN TPML_DIGEST_VALUES  *DigestList
-  );
-
-// MU_CHANGE - START - [TCBZ3267] Additional helper functions.
-
-/**
-  Get TPML_DIGEST_VALUES data size from HashAlgorithmMask
-
-  @param[in]     DigestList    TPML_DIGEST_VALUES data.
-
-  @return TPML_DIGEST_VALUES data size.
-**/
-UINT32
-EFIAPI
-GetDigestListSizeFromHashAlgorithmMask (
-  IN UINT32  HashAlgorithmMask
-  );
-
-// MU_CHANGE - END - [TCBZ3267] Additional helper functions.
-
-/**
-  This function get digest from digest list.
-
-  @param[in]  HashAlg       Digest algorithm
-  @param[in]  DigestList    Digest list
-  @param[out] Digest        Digest
-
-  @retval EFI_SUCCESS       Digest is found and returned.
-  @retval EFI_NOT_FOUND     Digest is not found.
-**/
-EFI_STATUS
-EFIAPI
-GetDigestFromDigestList (
-  IN TPMI_ALG_HASH       HashAlg,
-  IN TPML_DIGEST_VALUES  *DigestList,
-  OUT VOID               *Digest
-  );
-

```

Comment:
Need MU_CHANGE tags and for removed lines we comment them out (until the edk2 cherry-pick back)